projects/use/useop-qwt-config.cmake

```diff
@@ -0,0 +1,28 @@
+# Get the path to the overseer pro install
+get_filename_component(OP_ROOTDIR ${CMAKE_CURRENT_LIST_DIR}/../.. ABSOLUTE)
+get_filename_component(OP_ROOTDIR ${OP_ROOTDIR} ABSOLUTE)
+
+unset(QWT_INCLUDE_DIR CACHE)
+unset(QWT_LIBRARY_DIR CACHE)
+
+set(QWT_INCLUDE_DIR ${OP_ROOTDIR}/include/qwt)
```

Comment:
when including qwt files, do you want it to look like #include <qwt/header.hpp>  or #include <header.hpp>?